AEPAnalytics/Tests/UnitTests/AnalyticsAPITests.swift

```diff
@@ -202,7 +202,7 @@ class AnalyticsAPITests: XCTestCase {
         // test
         Analytics.getTrackingIdentifier(){(identifier, error) in
             XCTAssertNil(identifier)
-            XCTAssertEqual(error as? AEPError, .unexpected)
+            XCTAssertNil(error)
```

Comment:
We can update testGetTrackingIdentifier_IncorrectResponse to handle payload with non String values for example and create new tests for 
testGetTrackingIdentifier_correctResponseWithNilIdentifier()
Same comment for visitorIdentifier API.